libexec/cli/pull.exec

```diff
@@ -161,24 +152,18 @@ case "$SINGULARITY_CONTAINER" in
         SINGULARITY_WRITABLE=1
         export SINGULARITY_WRITABLE
 
-        if [ -z "${SINGULARITY_NOSUID:-}" -a -u "$SINGULARITY_libexecdir/singularity/bin/create-suid" ]; then
-            eval_abort "$SINGULARITY_libexecdir/singularity/bin/create-suid"
-        elif [ -x "$SINGULARITY_libexecdir/singularity/bin/create" ]; then
-            eval_abort "$SINGULARITY_libexecdir/singularity/bin/create"
-        else
-            message ERROR "Could not locate the Singularity binary: $SINGULARITY_libexecdir/singularity/bin/create\n"
-            exit 1
-        fi
+        "$SINGULARITY_bindir"/singularity image.create -s "$SINGULARITY_IMAGESIZE" "$SINGULARITY_IMAGE"
```

Comment:
This is probably just style / preference and has no functional difference, but I"d normally do like:

```
"${SINGULARITY_bindir}/singularity"
```
so the distinction of the variable is clear, in case there is absolutely any chance the quote could be confused with the variable, and the path is part of the same (enclosed) string.

Reply:
@vsoch ++